### ISSUE TITLE
Please update configuration display command for "ruckus_fastiron" as "show running-configuration"

### ISSUE THREAD

**Issue by mailsanjayhere** [ktbyers/netmiko#3445]
### Description of Issue/Question

Please update configuration display command for "ruckus_fastiron" as "show running-configuration" within utilities.py

### Netmiko device_type (if relevant to the issue)
ruckus_fastiron

Below is the output from the actual device..
![image](https://github.com/ktbyers/netmiko/assets/33064356/f9bfd1f2-2386-4b28-ae86-0b24c58c22cc)

```

**Comment by yearcode** [comment]
zte and ruijie  are show running-config

**Comment by ktbyers** [comment]
@yearcode @mailsanjayhere Yep, just submit pull-requests on both of these (if you want them).
